<launch>
	<param name="port_path" 
		type="str" 
		value="/dev/ttyS0" />
	<param name="node_type" 
		type="str" 
		value="mis_det_serial_node" />
	<param name="rate_Hz" 
		type="int" 
		value="10" />
	<node name="mis_det_serial_node"
        pkg="serial_comm" 
        type="serial_node.py" />
</launch>
